$MirOS: ports/games/frozen-bubble/patches/patch-server_Makefile,v 1.2 2006/11/04 04:12:03 tg Exp $
--- server/Makefile.orig	Sun Nov 12 19:55:39 2006
+++ server/Makefile	Sat Dec 23 01:36:22 2006
@@ -1,6 +1,6 @@
 include ../settings.mk
 
-CFLAGS ?= -g -Wall -Werror -pipe
+CFLAGS += -Wall -Werror
 CPPFLAGS += -I. `pkg-config glib-2.0 --cflags` -DVERSION=\"$(shell cat ../VERSION)\"
 LDLIBS = `pkg-config glib-2.0 --libs`
 
